### DISCUSSION TITLE
Filtro y separación

### DISCUSSION THREAD

**Discussion by decisoft** [PucelaBits/websegura#65] [Ideas]
¡Hola!

Mientras estaba revisando páginas para añadir, y mirando otras provincias, me vino algo a la cabeza a la hora de poner las webs de los ayuntamientos. Si vamos ampliando la base de datos de sitios webs, los ayuntamientos (por ejemplo) serán bastantes por provincia, ya sean solo los principales o algunos más. ¿Habría alguna forma de establecer un filtro de búsqueda para que el ciudadano/usuario pudiese buscar directamente su municipio (o cualquier otra institución)?

Saludos!

**Comment by nukeador** [comment]
De momento hicimos la separación por provincias para no volvernos muy locos, sobre todo porque los municipios pequeños normalmente solo tienen una web.

Lo que sí se planteo es poder poner etiquetas o categorías a los sitios para poder buscar por tipos de webs institucionales. Aunque más fácil que eso podría ser tener una búsqueda entre todas las webs que tenemos tipo "Ayuntamiento de Villarriba" y que te llevara al detalle de esta.

@palmerabollo había estado pensando en mejores filtros.
